readmes/download_attachments.md

```diff
@@ -0,0 +1,30 @@
+## Download Attachments from Feature Layer
+
+A user may want to download attachments that have been collected by their mobile workforce in ArcGIS Field Maps or ArcGIS Workforce. 
+
+This script allows users to download (either all or a subset) attachments from a particular feature layer or item to a folder on their computer.
+
+You can either specify an item id or feature layer url to use for this particular script.
+
+Supports Python 3.6+
+
+----
+
+In addition to the authentication arguments, the script specific arguments are as follows:
+
+- -item-id (Optional) - The item ID with the layer or table whose attachments you'd like to download
+- -layer-url (Optional) - The feature layer url where you'd like to download attachments
```

Comment:
The description of this is a little confusing. Should it be `...whose attachments you'd like to download` too?